pkg/agent/nodeportlocal/npl_agent_init.go

```diff
@@ -17,31 +17,29 @@
 package nodeportlocal
 
 import (
-	"context"
 	"errors"
 	"fmt"
 	"time"
 
-	"github.com/vmware-tanzu/antrea/pkg/agent/nodeportlocal/k8s"
+	nplk8s "github.com/vmware-tanzu/antrea/pkg/agent/nodeportlocal/k8s"
 	"github.com/vmware-tanzu/antrea/pkg/agent/nodeportlocal/portcache"
 	"github.com/vmware-tanzu/antrea/pkg/agent/nodeportlocal/util"
 
-	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/informers"
+	coreinformers "k8s.io/client-go/informers/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 )
 
-const resyncPeriod = 60 * time.Minute
+const resyncPeriod = 0 * time.Minute
```

Comment:
I think it would be good to explain why we choose 0 with a comment for folks not familiar with `resyncPeriod`. Maybe something like:
```go
// disable periodic calls to the UpdateFunc handler for existing objects. UpdateFunc will only be called when an object is actually updated.
```